ballerina/utils.bal

```diff
@@ -0,0 +1,231 @@
+// Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
```

Comment:
@PasinduDilmin  It seems like the license headers in the client files don't have the auto-generated clause. 

If we used `docs/license.txt` for the OpenAPI command, it should get generated correctly. shall we do that?